client/my-sites/domains/domain-management/dns/add-dns-record.jsx

```diff
@@ -0,0 +1,202 @@
+		this.onRestoreError = this.onRestoreError.bind( this );
+	}
+
+	onRestoreSuccess() {
+		const { translate, selectedDomainName } = this.props;
+		this.props.fetchDns( selectedDomainName, true );
+		this.props.successNotice(
+			translate( 'Yay, the name servers have been successfully updated!' )
+		);
+	}
+
+	onRestoreError( errorMessage ) {
+		this.props.errorNotice( errorMessage );
+	}
+
+	renderDnsTemplates() {
+		const selectedDomain = getSelectedDomain( this.props );
+
+		if ( ! selectedDomain || ! isMappedDomain( selectedDomain ) ) {
+			return null;
+		}
+
+		return (
+			<VerticalNav>
+				<DnsTemplates selectedDomainName={ this.props.selectedDomainName } />
+			</VerticalNav>
+		);
+	}
+
+	renderBreadcrumbs() {
+		const { translate, selectedSite, currentRoute, selectedDomainName } = this.props;
+
+		const items = [
+			{
+				label: translate( 'Domains' ),
+				href: domainManagementList( selectedSite.slug, selectedDomainName ),
+			},
+			{
+				label: selectedDomainName,
+				href: domainManagementEdit( selectedSite.slug, selectedDomainName, currentRoute ),
+			},
+			{
+				label: translate( 'DNS records' ),
+				href: domainManagementDns( selectedSite.slug, selectedDomainName ),
+			},
+			{
+				label: translate( 'Add a record', { comment: 'DNS record' } ),
+			},
+		];
+
+		const mobileItem = {
+			label: translate( 'Back to DNS records', {
+				comment: 'Link to return to the DNs records management page of a domain ',
+			} ),
+			href: domainManagementDns( selectedSite.slug, selectedDomainName ),
+			showBackArrow: true,
+		};
+
+		return (
+			<Breadcrumbs items={ items } mobileItem={ mobileItem } buttons={ [] } mobileButtons={ [] } />
+		);
+	}
+
+	goBack = () => {
+		const { selectedSite, selectedDomainName } = this.props;
+		page( domainManagementDns( selectedSite.slug, selectedDomainName ) );
+	};
+
+	renderMain() {
+		const { dns, selectedDomainName, translate } = this.props;
+		const dnsSupportPageLink = (
+			<ExternalLink
+				href={ localizeUrl( 'https://wordpress.com/support/domains/custom-dns/' ) }
+				target="_blank"
+				icon={ false }
+			/>
+		);
+		const mobileSubtitleText = translate(
+			'Add a new DNS record to your site. {{supportLink}}Learn more{{/supportLink}}',
+			{
+				components: {
+					supportLink: dnsSupportPageLink,
+				},
+			}
+		);
+		const explanationText = translate(
+			'Custom DNS records allow you to connect your domain to third-party services that are not hosted on WordPress.com, such as an email provider. {{supportLink}}Learn more{{/supportLink}}.',
+			{
+				components: {
+					supportLink: dnsSupportPageLink,
+				},
+			}
+		);
+
+		return (
+			<Main wideLayout className="add-dns-record">
+				<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
+				<div className="add-dns-record__fullwidth">
+					{ this.renderBreadcrumbs() }
+					<FormattedHeader
+						brandFont
+						headerText={ translate( 'Add a new DNS record' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Add new DNS record' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/add-dns-record-page)